### PR TITLE
OCPQE-11218: Append IDP for cucushift testing

### PR DESCRIPTION
**Pull request by liangxia**
Append a new IDP for cucushift testing, so IDPs provided by other tests won't be overwrite here.

### CODE REVIEW

File: ci-operator/step-registry/rosa/idp/htpasswd/rosa-idp-htpasswd-commands.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ IDP_PASSWD=$(openssl rand -base64 15)
 rosa create idp -c ${CLUSTER_ID} \
                 -y \
                 --type htpasswd \
-                --name htpasswd-1 \
+                --name rosa-htpasswd \
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/idp/htpasswd/rosa-idp-htpasswd-commands.sh:47]: I am running the steps, let me do this update in my PR https://github.com/openshift/release/pull/30943

Comment (liangxia) [reply ci-operator/step-registry/rosa/idp/htpasswd/rosa-idp-htpasswd-commands.sh]: Done. The change has been take out from this PR.

**Pull request merged**